mat/list_test.go

```diff
@@ -848,6 +866,10 @@ func makeCopyOf(a Matrix) Matrix {
 		}
 		copy(d.mat.Data, diag.mat.Data)
 		return returnAs(d, t)
+	case *Tridiag:
+		var m Tridiag
+		m.Clone(a.(*Tridiag))
```

Comment:
Sorry I missed this before, but this looks wrong; on `*Dense` we have `CloneFrom` and on `*VecDense` we have `CloneFromVec`. This seems likely to be `CloneFromTridiag`?

We unfortunately also have a residual `Clone` on `*Cholesky`. Maybe we should have a discussion about how to deal with these properly.

Reply:
Yes, `Cholesky.Clone` is what I followed. The other `CloneFrom` methods take an interface but `Tridiag` has no corresponding interface besides `Banded`. What about `Tridiag.CloneFromBand` which would panic if the input's bandwidth is not `1,1`?

Reply:
I'm not sure, I don't like the idea of losing type safety when we don't need to. Maybe clone methods that take a concrete type should name the concrete type, so `Cholesky.Clone` => `Cholesky.CloneFromCholesky` and `Tridiag.Clone` => `Tridiag.CloneFromTriDiag`. They seem heavy though, and there is no risk of additional interface collision with `ClonerFrom` (which they could also be renamed to because they won't ever reasonably need to implement that), but the source would be more explicit.

Reply:
SGTM and done. I'll send a PR for Cholesky.Clone.